src/OrderLinesList/Details/OrderLineDetails.js

```diff
@@ -92,6 +92,17 @@ const OrderLineDetails = ({
     [lineId, order],
   );
 
+  const goToReceive = useCallback(
+    () => {
+      history.push({
+        pathname: '/receiving',
+        search: `qindex=poLine.poLineNumber&query=${line?.poLineNumber}`,
+      });
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [lineId, line],
+  );
```

Comment:
We usually write hooks in style

```
useHook(() => {
  ..
}, [deps])
```

Let's keep `history` in deps too, just to let the linter do his job. `histrory` has the same ref in memory, so it will not cause extra rerenders

```suggestion
  const goToReceive = useCallback(() => {
    history.push({
      pathname: '/receiving',
      search: `qindex=poLine.poLineNumber&query=${line?.poLineNumber}`,
    });
  }, [history, lineId, line?.poLineNumber]);
```